src/screens/HtmlScreen.js

```diff
@@ -22,7 +29,7 @@ export const HtmlScreen = ({ navigation, route }) => {
   const [refreshing, setRefreshing] = useState(false);
   const trackScreenViewAsync = useTrackScreenViewAsync();
 
-  if (!query || !queryVariables?.name) return null;
+  if (!query || !queryVariables?.name) return <EmptyMessage title={texts.empty.content} />;
```

Comment:
we are missing screen wrapping components here. compared to the the "main" return of this component, there are following wrappers we should add here:

- `<SafeAreaViewFlex>`
- `<WrapperWithOrientation>`
- `<Wrapper>`

this way, the empty message should be placed more beautiful on the screen.
can you give it a try? otherwise i will dig deeper before merging the branch once the next release will be drafted.

Reply:
I didn't know I had to write it this way. I'm updating the code right now.